src/components/Notifications.js

```diff
@@ -99,7 +99,7 @@ const NotificationDisplay = Utils.connectStore(notificationStore, 'notificationS
           }, ['Details'])
         ]),
         h(Clickable, {
-          'aria-label': 'Dismiss notification',
+          'aria-label': type ? `Dismiss ${type} notification` : 'Dismiss notification',
```

Comment:
TOL: This is fine as a short-term fix, but going forward we should look into improving the accessibility of notifications, which will also let us write a more natural selector in the tests.